src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllCodeFixSuggestedAction.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.UnifiedSuggestions.UnifiedSuggestedActions;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    /// <summary>
+    /// Suggested action for fix all occurrences code fix.  Note: this is only used
+    /// as a 'flavor' inside CodeFixSuggestionAction.
+    /// </summary>
+    internal sealed partial class FixAllCodeFixSuggestedAction : FixAllSuggestedAction, ITelemetryDiagnosticID<string>, IFixAllCodeFixSuggestedAction
```

Comment:
Existing code: https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllSuggestedAction.cs.
Renamed `FixAllSuggestedAction` to `FixAllCodeFixSuggestedAction` and moved some code to the base type.

Reply:
nit: can we rename FixAllSuggestedAction to AbstractFixAllSuggestedAction?